.github/workflows/publish-docs.yaml

```diff
@@ -14,80 +14,151 @@ jobs:
+        # Clean the working copy
+        git clean -dxf
+
+    - name: Checkout master
+      working-directory: docs
+      run: |
+        git reset origin/master --hard
+        sed -i 's|head-protocol|head-protocol/unstable|' docusaurus.config.js
+
+    - name: 📥 Download latest hydra-spec
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        repo: input-output-hk/hydra
         workflow: ci-nix.yaml
         workflow_conclusion: success
         branch: master
-        name: docs-unstable
-        path: docs-unstable
+        name: hydra-spec
+        path: docs/static
 
-    - name: 📥 Download latest / docs
+    - name: 📥 Download latest benchmarks
       uses: dawidd6/action-download-artifact@v2
       with:
+        repo: input-output-hk/hydra
         workflow: ci-nix.yaml
         workflow_conclusion: success
         branch: master
-        name: docs
-        path: docs-master
+        name: benchmarks
+        path: docs/benchmarks
 
-    - name: 🪓 Piece together docs
+    - name: 📥 Download latest test-results
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        repo: input-output-hk/hydra
+        workflow: ci-nix.yaml
+        workflow_conclusion: success
+        branch: master
+        name: test-results
+        path: docs/benchmarks/tests
+
+    - name: 📥 Download latest haddock documentation
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        repo: input-output-hk/hydra
+        workflow: ci-nix.yaml
+        workflow_conclusion: success
+        branch: master
+        name: haddocks
+        path: docs/static/haddock
+
+    - name: Build /unstable documentation
+      working-directory: docs
       run: |
-        mkdir public
-        mv docs-release public/head-protocol
-        mv docs-unstable public/head-protocol/unstable
-        # Always use monthly reports from master
-        # XXX: This depends on languages and assets are annoying
-        rm -r public/head-protocol/monthly
-        cp -r docs-master/monthly public/head-protocol/monthly
-        rm -r public/head-protocol/ja/monthly
-        cp -r docs-master/ja/monthly public/head-protocol/ja/monthly
-        rm -r public/head-protocol/fr/monthly
-        cp -r docs-master/fr/monthly public/head-protocol/fr/monthly
-        # XXX: Need to copy assets as well. This will also litter with unrelated assets (js)
-        cp -r docs-master/assets/* public/head-protocol/assets/
-        cp -r docs-master/ja/assets/* public/head-protocol/ja/assets/
-        cp -r docs-master/fr/assets/* public/head-protocol/fr/assets/
+        yarn
+        yarn build
+
+        mv build /tmp/public/head-protocol/unstable
 
     - name: 👉 Create redirect
       run: |
-        echo "hydra.family" > public/CNAME
-        echo '<!DOCTYPE html><html><head><meta http-equiv="Refresh" content="0; URL=https://hydra.family/head-protocol"></head></html>' > public/index.html
+        echo "hydra.family" > /tmp/public/CNAME
```

Comment:
Question: Why do we need this change?

Reply:
I am cleaning the working copy in a step before. Hence, want to assemble the website somewhere outside. Opted for `/tmp/`